README.md

```diff
@@ -1,23 +1,25 @@
-# InfluxData Helm Charts
+# InfluxData Helm charts
 
 [![License](https://img.shields.io/badge/license-MIT-green.svg)](./LICENSE)
 [![](https://github.com/influxdata/helm-charts/workflows/helm-charts%2Frelease/badge.svg?branch=master)](https://github.com/influxdata/helm-charts/actions)
```

Comment:
can we add `[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.influxdata.com/slack)` here?

Reply:
@thanks, @russorat - added